images/skopeo-docker-io.yaml

```diff
@@ -99,10 +99,8 @@ docker.io:
     prom/cloudwatch-exporter:
       - "cloudwatch_exporter-0.4"
     prom/prometheus:
-      - "v2.6.1"
-      - "v2.9.1"
-      - "v2.13.1"
-      - "v2.15.2"
+      - ">= 2.30.0"
+      - ">= 2.43.0-stringlabels"
```

Comment:
IIRC, the ">=" operator splits the version number, so it may require a "simple" version and may not work with `-stringlabels`

Reply:
Oh, forget about it. We've moved to skopeo since then.